baseplate/sidecars/live_data_watcher.py

```diff
@@ -32,6 +36,41 @@ def __init__(self, dest: str, owner: int, group: int, mode: int):
         self.group = group
         self.mode = mode
 
+    @staticmethod
+    def fetch_data_from_url(url: str) -> Optional[str]:
+        data = None
+        try:
+            # Fetch the data from the url as bytes.
+            data = requests.get(url).content
+        except requests.exceptions.RequestException as e:
+            logger.exception(e)
+            return None
+        return data
+
+    @staticmethod
+    def get_data_to_write(json_data: dict, data: bytes) -> Optional[bytes]:
+        # Check if we have a JSON in this special format:
+        # data = {
+        #    "live_data_watcher_load_type": str
+        #    "data": str
+        #    "md5_hashed_data": str
```

Comment:
why using md5 in 2022? also I don't see this actually checked in the code

Reply:
I should probably remove this from the comment here in the code, but the point of the `md5_hashed_data` is just for clients to know that it's a new version of the ZK file. Otherwise the contents of the ZK file will always look identical even when we re-write to it which makes debugging quite a bit more difficult. I don't think baseplate will ever care about this field.

Reply:
Updated to new approach.